app/resources/v1/pacbio/container_material_resource.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
```

Comment:
Holy crap!!! I didn't realise you would have to do this. There is a lot of complexity to make this work which almost negates the point of polymorphism. We definitely need to move PacBio over to GraphQL. Hopefully this is an interim measure. Again could you get Harry to look at this as well.